auth-web/src/routes/router.ts

```diff
@@ -372,6 +374,13 @@ export function getRoutes (): RouteConfig[] {
       props: true,
       meta: { requiresAuth: false }
     },
+    {
+      path: '/make-pad-payment/:paymentId/transactions/:redirectUrl',
```

Comment:
can we have a better route? Technically, this is not a PAD payment.. may be make-cc-payment ?

Reply:
yup..wil change it

Reply:
done